.travis.yml

```diff
@@ -95,6 +94,7 @@ script:
   - python -c "import sys; print('\n'.join(sys.path))"
   - mkdir build
   - pushd build
+  - ulimit -n 2048
```

Comment:
Is this required at compile time or when running the tests?
We had issues about "too many files open" when using libres from everest (it was the first usage where a lot of simulations were run on the same ert instance). And the fix was to hunt down the files that were left open. 
If this is required for compiling, it's fine, if it is for running the tests, then maybe we should also open an issue saying that there are still files left open somewhere

Reply:
This is for running the some of the tests. The default is 256 on osx and 1024 on linux